### PR TITLE
docs(modal/dialog-box): [modal,dialog-box] update docs

### PR DESCRIPTION
**Pull request by James-9696**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [* ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [* ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ *] No

### CODE REVIEW

File: examples/sites/demos/pc/app/dialog-box/basic-usage-composition-api.vue [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <template>
   <div>
-    <tiny-button @click="boxVisibility = true" :title="'弹出 Dialog' + boxVisibility">弹出 Dialog</tiny-button>
+    <tiny-button @click="boxVisibility = true" title="弹出 Dialog">弹出 Dialog</tiny-button>
```
Comment (zzcr) [inline examples/sites/demos/pc/app/dialog-box/basic-usage-composition-api.vue:3]: 改了demo对应的E2E测试用例也需要修改吧

Comment (James-9696) [reply examples/sites/demos/pc/app/dialog-box/basic-usage-composition-api.vue:3]: 已修改

File: examples/sites/demos/pc/app/components.js [hunk 2]
```diff
@@ -493,7 +493,7 @@ export default {
     },
     Methods: {},
     Slots: {
-      default: 'value',
+      default: 'grid-default',
```
Comment (gimmyhehe) [inline examples/sites/demos/pc/app/components.js]: 目前components.js已经用不到，此处改动的具体原因是什么~

Comment (James-9696) [reply examples/sites/demos/pc/app/components.js]: 调试后未及时还原 已还原 此处无须修改

**Review by zzcr — APPROVED** [review]

**Pull request merged**